### DISCUSSION TITLE
Create a preview tab in gist files if the format is markdown

### DISCUSSION THREAD

**Discussion by btzo** [community/community#4481] [Other Feature Feedback, Questions, & Ideas] [answered]
The site [gist.github.com](https://gist.github.com/) allows you to create gists of various types.

In case a file is in markdown format, it would be convenient if the web interface could automatically detect it and provide a preview tab.

This functionality already exists today when we make comments on Pull Requests and/or start discussions, the goal was to have something in the same format.

[Several](https://gistpreview.github.io/) [alternative](https://github.com/revathskumar/gist-markdown-preview) [solutions](https://chrome.google.com/webstore/detail/gist-markdown-preview/icgfmklfpokeihbnnjjdpnjndickgpkh?hl=en) exist on the Internet, which shows that the demand exists.

**Comment by davidstaheli** [comment, marked as answer]
@btzo, this is now complete and described in [this changelog](https://github.blog/changelog/2021-11-17-preview-the-markdown-rendering-of-gists/). Thank you for this suggestion!

**Reply by btzo** [reply]
Thank you for finally bringing this functionality !
